app/logic/Mail/EWS/EWSAccount.ts

```diff
@@ -61,6 +59,12 @@ export class EWSAccount extends MailAccount {
       await this.oAuth2.login(interactive);
     }
     await this.listFolders();
+  }
+
+  async login(interactive: boolean): Promise<void> {
+    await ensureLicensed();
+    await super.login(interactive);
+    await this.verifyLogin(interactive);
```

Comment:
Please make another function for that (which will be called by `login()` and `verifyLogin()`). Don't re-use `verifyLogin()`